### ISSUE TITLE
Verification and evaluation of code

### ISSUE THREAD

**Issue by WHK1229** [lyuwenyu/RT-DETR#403]
How to output datasets with evaluation accuracy of P, R, mAP, FPS, Gflops

**Comment by lyuwenyu** [comment]
1. You should modified `coco_eval` file
> https://github.com/lyuwenyu/RT-DETR/blob/main/rtdetrv2_pytorch/src/data/dataset/coco_eval.py

2. You can use `torch.profiler`
> https://pytorch.org/docs/stable/profiler.html

**Comment by WHK1229** [comment]
直接在命令行运行python /home/whk/hdd/RT-DETR-main/rtdetr_pytorch/src/data/coco/coco_eval.py     没有反应，也不报错

**Comment by lyuwenyu** [comment]
```bash

CUDA_VISIBLE_DEVICES=0,1,2,3 torchrun --master_port=9909 --nproc_per_node=4 tools/train.py -c path/to/config -r path/to/checkpoint --test-only

```

**Comment by bshakya77** [comment]
Hello @lyuwenyu,

I have trained rtdetrv2_r50vd model on VisDrone2019 dataset. How can I calculate the GFLOPs and Inference time in test-dev dataset. The official page states that GGLOPs of rtdetrv2_r50vd as 136 when trained on COCO dataset with 640 image size. Does the GFLOPs changes with dataset used?

If you have any reference to calculate the GLOPs and inference time, it would be helpful.

Thank you.

Regards,
Bijay
